homeassistant/components/synology_srm/device_tracker.py

```diff
@@ -37,6 +37,34 @@
     }
 )
 
+ATTRIBUTE_ALIAS = {
+    "band": None,
+    "connection": None,
+    "current_rate": None,
+    "dev_type": None,
+    "hostname": None,
+    "ip6_addr": None,
+    "ip_addr": None,
+    "is_baned": None,
```

Comment:
Is this a spelling error here or in the api? `baned`-> `banned`

Reply:
Well spotted ... those actually come directly from the router ... guess I should alias it to be the correct spelling?

Reply:
Yes, please! :+1: